exchange-contract/docs/notebooks/factories/token-issuer.ipynb

```diff
@@ -107,29 +107,15 @@
    },
    "outputs": [],
    "source": [
-    "instance_directory = os.path.join(notebook_directory, 'instances', token_class)\n",
-    "instance_file = os.path.join(instance_directory, 'issuer.ipynb')\n",
-    "template_file = os.path.join(notebook_directory, 'templates', 'token-issuer.ipynb')\n",
+    "instance_parameters = {\n",
+    "    'token_owner' : identity,\n",
+    "    'token_class' : token_class,\n",
+    "    'service_host' : service_host,\n",
+    "    'notebook_directory' : notebook_directory,\n",
```

Comment:
this is really a comment on the "Configure Token information" Cell. Given that we already specify service_host as ENV, do we still need service_host parameter to be explicitly specified? 

Also, there is something wierd with this Configure Token information Cell where you enter all the info. Before this PR, after the last info (service_host) when I press ENTER, the execution cursor automatically goes to the next cell, (as expcted), but now, it comes back to the same cell, and when i press "play button", i am asked to enter all the info all over again. Of course, I can manually move the play button to the next cell, that works.